### DISCUSSION TITLE
Does the nRF52840 Dongle require sudo

### DISCUSSION THREAD

**Discussion by rahav** [contiki-ng/contiki-ng#2752] [Q&A] [answered]
For some reason I am able to view the COM port of a DK without `sudo` but the Dongle requires `sudo`. Is that expected?

The devices are all connected to the same computer, running Ubuntu 22, and we are using `picocom`.

**Comment by pjonsson** [comment, marked as answer]
No, it's the permissions of the serial port that matter. I don't have a DK, but the dongle works fine for me without sudo.

I haven't used this myself, but you might try https://github.com/NordicSemiconductor/nrf-udev
